LibraryApp/src/app/components/Book/styles.js

```diff
@@ -0,0 +1,36 @@
+import { StyleSheet } from 'react-native';
+import { FONT_SIZE_LARGE, FONT_SIZE_MEDIUM } from '../../../constants/dimensions';
+import { white } from '../../../constants/colors';
+
+const IMAGE_SIZE = 80;
```

Comment:
I would not use constants for numbers that are used only once. However, I like having all the constants together. I will let @WFolini decide on this